managed/src/test/java/com/yugabyte/yw/models/InstanceTypeTest.java

```diff
@@ -29,6 +31,9 @@
   private Customer defaultCustomer;
   private InstanceType.InstanceTypeDetails defaultDetails;
 
+  @Mock
```

Comment:
where is a test where you have to setup this mock?
`when(mockConfig.get(..._COUNT)).thenReturn(1);`
`when(mockConfig.get(..._GB)).thenReturn(250);`

Reply:
Ah! Didn't realise this is for entity class. We can't mock config in this file. Most of the values are being read from in memory.
Instead used ConfigFactory.load() just to test our code. I think, this should be fine if not please suggest! 
Also due to static method used in Entity class, getting difficulties in writing Unit test cases, need to refactor the code(Entity class). But for now I think this should be fine. 

Thanks!